src/Command/Vendors/VendorLoadCommand.php

```diff
@@ -84,15 +105,23 @@ protected function execute(InputInterface $input, OutputInterface $output): int
 
         $results = [];
         foreach ($vendorServices as $vendorService) {
+            $labels = [
+                'type' => 'vendor',
+                'vendorName' => $vendorService->getVendorName(),
+                'vendorId' => $vendorService->getVendorId(),
+            ];
+
             try {
                 /* @var VendorServiceInterface $vendorService */
                 $vendorService->setWithoutQueue($dispatchToQueue);
-                $vendorService->setWithUpdates($withUpdates);
+                $vendorService->setWithUpdatesDate($withUpdatesDate);
```

Comment:
`$withUpdatesDate` has no default value, so can be `null`, assuming we fix it so that it really is `InputOption::VALUE_OPTIONAL`